src/utils/normalize-request.ts

```diff
@@ -1,47 +1,37 @@
-import { Config } from '../config'
+import { Config } from '../config';
 
-const MEDIA_FILE_EXTENSIONS = 'css gif ico jpg js otf jpeg png svg ttf webp woff woff2 csv json'.split(' ')
-const hasMediaFileExtension = (path: string): boolean => {
-  const ext = path.split('.').pop()?.toLowerCase()
-  return ext ? MEDIA_FILE_EXTENSIONS.includes(ext) : false
-}
+const MEDIA_FILE_EXTENSIONS = new Set('css gif ico jpg js otf jpeg png svg ttf webp woff woff2 csv json'.split(' '));
```

Comment:
```suggestion
const MEDIA_FILE_EXTENSIONS = new Set([
  'css',
  'csv',
  'gif',
  'ico',
  'jpeg',
  'jpg',
  'js',
  'json',
  'otf',
  'png',
  'svg',
  'ttf',
  'webp',
  'woff',
  'woff2',
]);```

(1) no need to actually pass a string and use `.split` on it
(2) 1 line per item in an array should be the default code style. It causes the simplest git-diffs when adding/removing items (trailing comma!), and we can sort items by ABC making it very easy to visually scan for a specific element